package.json

```diff
@@ -44,8 +44,8 @@
     "pre-commit:lint": "yarn lint-staged",
     "pre-commit:types": "yarn tsc",
     "pre-commit:test": "yarn test",
-    "test": "cross-env CI=1 react-scripts test --env=jsdom",
-    "test:watch": "react-scripts test --env=jsdom",
+    "test": "cross-env CI=1 react-scripts test --env=jsdom-fourteen",
+    "test:watch": "react-scripts test --env=jsdom-fourteen",
```

Comment:
What is this change?

Reply:
Oh, when I was trying to test with `waitFor`, I discovered the react-scripts default version of jsdom doesn't support a required mutation observer.

However, I don't have `waitFor` in the tests anymore, so I'm reverting this.